policy/README.md

```diff
@@ -365,6 +389,280 @@ If a vehicle is matched with a rule, then it _will not_ be considered in the sub
+
+You may also show which APIs, endpoints, and fields your agency is serving to providers and the public. This is an `available_apis` array within the [Requirement Data Specs](#requirement-data-specs) section in the [Requirement](#requirement) data feed.
+
+```jsonc
+// ...  
+          "required_apis": [
+            {
+              "api_name" : "[API NAME]",
+              "required_endpoints": [ 
+                {
+                  "endpoint_name" : "[ENDPOINT NAME]",
+                  "required_fields": [
+                    "[FIELD NAME]",
+                    // other field names
+                  ],
+                  "disallowed_fields": [
+                    "[FIELD NAME]",
+                    // other field names
+                  ]
+                },
+                // other endpoints
+              ]
+            },
+            // other APIs in same data spec
+          ],
+          "available_apis": [
+            {
+              "api_name" : "[API NAME]",
+              "available_endpoints": [ 
+                {
+                  "endpoint_name" : "[ENDPOINT NAME]",
+                  "url": "[ENDPOINT URL]",
+                  "available_fields": [
+                    "[FIELD NAME]",
+                    // other field names
+                  ]
+                },
+                // other endpoints
+              ]
+            }
+          ]
+// ...
+```
+
+| Name                 | Type  | Required/Optional | Description                | 
+| -------------------- | ----- | -------- | ----------------------------------- | 
+| `api_name`           | Text  | Required | Name of the applicable API required. At least one API is required. APIs not listed will not be available to the agency. E.g. for MDS: 'provider', or 'agency'. For GBFS, this field is omitted since GBFS starts at the `endpoint` level. | 
+| `endpoint_name`      | Text  | Required | Name of the required endpoint under the API. At least one endpoint is required. E.g. for MDS 'provider': 'trips' | 
+
+**Provider Endpoints** - Specific to the `required_apis` array 
+
+| Name                 | Type  | Required/Optional | Description                | 
+| -------------------- | ----- | -------- | ----------------------------------- | 
+| `required_endpoints` | Array | Required | Array of optional endpoints required by the agency. At least one is required. Endpoints not listed will not be available to the agency. | 
+| `required_fields`    | Array | Optional | Array of optional field names required by the agency. Can be omitted if no optional fields are required. Use dot notation for nested fields. See **special notes** below. | 
+| `disallowed_fields`  | Array | Optional | Array of optional field names who's values are to be returned by the agency as `null`, even if required in MDS. Use dot notation for nested fields. See **special notes** below. | 
```

Comment:
My main concern with removing the required field itself is that any schema validation will then fail, since the field is required. 

Of course, if there is also validation on the value of that field, then that would fail as well. So maybe it's not an issue per se. Language may have to be added to say that schema validations may fail if required fields are removed.

Reply:
Yep, I think that's a good path. I think maybe the validation _should_ fail - we're explicitly allowing Agencies to break the spec by disallowing required fields.

Reply:
I think that generally in MDS we've considered `null` equivalent to `undefined` by convention (I swear that there was some issue for this at one point), so we should make sure that this aligns. In a lot of relational databases, `undefined` is _persisted_ as `null`, so the semantic difference between "this was never available" versus "this was omitted when it was sent" is lost anyways.

Reply:
I've made an update in this [commit](https://github.com/openmobilityfoundation/mobility-data-specification/commit/648b2192b4b6eee9ab9b013c1e2f0a23ba68e669#diff-5dbb467da8c3ceb340b4c471496e28b63ed45d1ee3e15ceed7c44c3b88340720). Let me know what you think.

Reply:
lgtm!